includes/Modules/Analytics_4.php

```diff
@@ -213,6 +219,17 @@ public function __construct(
 		$this->custom_dimensions_data_available = new Custom_Dimensions_Data_Available( $this->transients );
 		$this->reset_audiences                  = new Reset_Audiences( $this->user_options );
 		$this->audience_settings                = new Audience_Settings( $this->options );
+		$this->audiences_util                   = new Audiences(
```

Comment:
We should simple pass `$this->audience_settings` to the `Audiences` constructor, as we do for `Resource_Data_Availability_Date` below, and use it within the class rather than creating this callback.